src/test/java/TestNG/TestSuite/DropDownValidation.java

```diff
@@ -23,7 +25,67 @@ public void DropdownListValidation(){
+
+    }
+    @Test
+    public void GetAllOptions(){
+        lpage.NavigateToUrl("https://demo.seleniumeasy.com/");
+        Assert.assertTrue(shomepage.VerifySeleniumeasyHomepage("Input Forms"),"homepage is not displayed");
+        shomepage.ClickonNavmenuitem("Input Forms");
+        shomepage.ClickonInputformitem("Select Dropdown List");
+        Assert.assertTrue(dpage.VerifySelectList("Select List Demo"),"Dropdown page is not displayed");
+        List<String> allOptions=dpage.GetDropdownList();
+        System.out.println("No.of days present in list is "+allOptions.size());
+        for(String option:allOptions)
+        {
+            System.out.println(option);
+        }
+
+    }
+    @Test
+    public void MultiSelectListValidation(){
+        lpage.NavigateToUrl("https://demo.seleniumeasy.com/");
+        Assert.assertTrue(shomepage.VerifySeleniumeasyHomepage("Input Forms"),"homepage is not displayed");
+        shomepage.ClickonNavmenuitem("Input Forms");
+        shomepage.ClickonInputformitem("Select Dropdown List");
+        Assert.assertTrue(dpage.VerifySelectList("Select List Demo"),"Dropdown page is not displayed");
+        String state="California";
+        dpage.SelectState(state);
+        dpage.ClickOnFirstSelected();
+        String expectedSelectState="First selected option is : "+state+"";
+        String actualSelectState=dpage.GetFirstSelectedOption();
+        Assert.assertEquals(actualSelectState,expectedSelectState);
+
+    }
+    @Test
+    public void MultiSelectListValidation1(){
+        lpage.NavigateToUrl("https://demo.seleniumeasy.com/");
+        Assert.assertTrue(shomepage.VerifySeleniumeasyHomepage("Input Forms"),"homepage is not displayed");
+        shomepage.ClickonNavmenuitem("Input Forms");
+        shomepage.ClickonInputformitem("Select Dropdown List");
+        Assert.assertTrue(dpage.VerifySelectList("Select List Demo"),"Dropdown page is not displayed");
+        String state="Texas";
+        dpage.SelectState(state);
+        dpage.ClickOnGetAllSelected();
+        String expectedGetAllSelectedstate="Options selected are : "+state+"";
+        String actualGetAllSelectedstate=dpage.OptionsSelected();
+        Assert.assertEquals(actualGetAllSelectedstate,expectedGetAllSelectedstate);
+        //Assert.assertTrue(dpage.VerifySelectedOption(state));
+    }
+    @Test
+    public void AllDropdownItems(){
+        lpage.NavigateToUrl("https://demo.seleniumeasy.com/");
+        Assert.assertTrue(shomepage.VerifySeleniumeasyHomepage("Input Forms"),"homepage is not displayed");
+        shomepage.ClickonNavmenuitem("Input Forms");
+        shomepage.ClickonInputformitem("Select Dropdown List");
+        Assert.assertTrue(dpage.VerifySelectList("Select List Demo"),"Dropdown page is not displayed");
+        System.out.println(dpage.GetDropdownList2());
+        System.out.println("No.of states present in list is "+dpage.GetDropdownList2().size());
```

Comment:
Validation steps not written means assert conditions whether all dropdown its are getting in actual result